bob-common/src/configs/node.rs

```diff
@@ -481,6 +481,8 @@ pub struct Node {
     #[serde(skip)]
     disks_ref: Arc<Mutex<Vec<DiskPath>>>,
 
+    #[serde(default = "Node::default_error_log_interval")]
+    error_log_interval: String,
```

Comment:
This parameter is redundant. It value will never change. It is better to remove the part related to the config and just declare a constant at the place of use